### DISCUSSION TITLE
Visual Editor - insert code syntax markdown block

### DISCUSSION THREAD

**Discussion by richardleestreamgo** [requarks/wiki#2947] [Help / Questions] [answered]
Hi

I read this is available v 2.5 but I have this version and can't find it? Can you help explain how to use visual editor to insert code blocks like slite?

Thanks

**Comment by NGPixel** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/15522395/105536674-b418d500-5cbe-11eb-85cc-b9d65d7691b5.png)
